var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -7,20 +7,33 @@
 
 
 class PyFenicsBasix(PythonPackage):
-    """Python basis evaluation library for Next generation FEniCS problem solving
-    environment"""
+    """Python interface to Basix, a finite element definition and tabulation runtime library"""
 
     homepage = "https://github.com/FEniCS/basix"
     url = "https://github.com/FEniCS/basix/archive/v0.1.0.tar.gz"
     git = "https://github.com/FEniCS/basix.git"
-    maintainers = ["chrisrichardson", "mscroggs", "garth-wells"]
+    maintainers = ["chrisrichardson", "mscroggs", "garth-wells", "jhale"]
 
     version("main", branch="main")
     version("0.5.1", sha256="69133476ac35f0bd0deccb480676030378c341d7dfb2adaca22cd16b7e1dc1cb")
     version("0.4.2", sha256="a54f5e442b7cbf3dbb6319c682f9161272557bd7f42e2b8b8ccef88bc1b7a22f")
-    version("0.3.0", sha256="9b148fd2a5485c94011fc6ca977ebdef0e51782a62b3654fc044f35b60e2bd07")
-    version("0.2.0", sha256="e1ec537737adb283717060221635092474e3f2b5b5ba79dfac74aa496bec2fcb")
-    version("0.1.0", sha256="2ab41fe6ad4f6c42f01b17a6e7c39debb4e0ae61c334d1caebee78b741bca4e7")
+    version(
+        "0.3.0",
+        sha256="9b148fd2a5485c94011fc6ca977ebdef0e51782a62b3654fc044f35b60e2bd07",
+        deprecated=True,
+    )
+    version(
+        "0.2.0",
+        sha256="e1ec537737adb283717060221635092474e3f2b5b5ba79dfac74aa496bec2fcb",
+        deprecated=True,
+    )
+    version(
+        "0.1.0",
+        sha256="2ab41fe6ad4f6c42f01b17a6e7c39debb4e0ae61c334d1caebee78b741bca4e7",
+        deprecated=True,
+    )
+
+    variant("numba", default=False, description="numba features")
```

Comment:
I don't think we want to make `numba` a variant. Users wanting `numba` can just concretize `py-fenics-basix` and `numba` together.